components/modal/modalContent.css

```diff
@@ -40,10 +41,6 @@
   border-radius: 0;
 }
 
-.ui-modal-content-block:last-of-type {
-  height: 100%;
```

Comment:
This increase height of last content block to fill left part modal window. Could you please check solution with adding `min-heigth` instead of removing `height`?

Reply:
fill left part modal window --> where can i check it? The height of the last block is stretched to the bottom, but it should be stretch by content inside

Reply:
hmm, this stretching made me confused. Because on some design we have it stretched to the bottom but on some to content inside. So, if we are going to content inside version - I'm ok with it :)

Reply:
@spzm we discussed with Gerard and the modal should wrap the content inside unlike the sidePanel. It is easy to get confused, and was not very clear on the designs also... Now the designs are updated, especially for small states.